docs/runningtests.md

```diff
@@ -20,7 +20,7 @@ Options:
   --reporter=<reporter>,   Select reporter: crash, dots, singleline, xunit,
                            info, spec
   --colorizer=<colorizer>, Select color theme: off, light, dark
-  --formatter=<formatter>, Select error formatter: posix, vs
+  --formatter=<formatter>, Select error formatter: posix, vs, qt
```

Comment:
This is not the actual bandit help output. Choices are sorted (posix, qt, vs). In fact, the whole `--help` in the documentation is outdated. Would you be so kind and update it in your PR?